lib/high_voltage/configuration.rb

```diff
@@ -46,9 +46,9 @@ def set_default_configuration
 
       self.content_path = 'pages/'
       self.home_page = nil
-      self.layout = 'application'
       self.route_drawer = HighVoltage::RouteDrawers::Default
       self.routes = true
+      self.parent_controller = 'AppliationController'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.